packages/react/src/TooltipV2/Tooltip.module.css

```diff
@@ -109,8 +110,8 @@
   }
 
   /* Animation styles */
-  &.\\:popover-open,
-  &.\\:popover-open::before {
+  /* stylelint-disable-next-line selector-class-pattern */
+  &.\\:popover-open, &:global(.\\:popover-open)::before {
```

Comment:
@jonrohan similar to https://github.com/primer/react/pull/5716/files#r1963911493, wrapped these in `:global`